pallets/api/src/fungibles/tests.rs

```diff
@@ -47,13 +47,17 @@ mod encoding_read_result {
 
 	#[test]
 	fn token_name() {
-		let name = vec![42, 42, 42, 42, 42];
+		let mut name = Some(vec![42, 42, 42, 42, 42]);
```

Comment:
`let name = "some name".as_bytes()` wouldve been nicer, more descriptive. Looping through an array of the cases might also neaten this up slightly

```
for name in [Some("Pop".as_bytes()), None]
{ assert_eq!(..) }
```

Out of scope, but using actual strings in other tests might be nicer, but not a requirement.